src/types/QIX.ts

```diff
@@ -35,10 +35,20 @@ interface NullValueRepresentation {
   text?: string;
 }
 
+export interface RowHeight {
+  linesCount: number;
+}
+
+export interface Component {
+  key: string;
```

Comment:
I think you should be explicit about the `key` here.

```suggestion
  key: "general" | "theme";
```